perf_test/blas/blas3/KokkosBlas3_trmm_perf_test.hpp

```diff
@@ -492,14 +503,17 @@ trmm_args_t __do_setup(options_t options, matrix_dims_t dim) {
   trmm_args.diag  = options.blas_args.trmm.trmm_args.c_str()[3];
   trmm_args.A     = vta("trmm_args.A", options.n, dim.a.m, dim.a.n);
   trmm_args.B     = vtb("trmm_args.B", options.n, dim.b.m, dim.b.n);
+  host_A = Kokkos::create_mirror_view(trmm_args.A);
```

Comment:
@e10harvey you should be able to create `host_A` via `auto` type deduction, I don't think the declaration at line 497 above is necessary unless you prefer this pattern (similar other comment)